lib/blog-encrypt.js

```diff
@@ -30,7 +30,24 @@ function decryptAES () {
       document.getElementById('encrypt-blog').style.display = 'inline';
       document.getElementById('encrypt-blog').innerHTML = '';
       // Use jquery to load some js code
-      $('#encrypt-blog').html(content);
+      try {
+        
+        $('#encrypt-blog').html(content);
+
+      } catch(e) {
+        
+        var errorInfo = "<p>"
+                      +   "You can see this mainly because there are some error "
```

Comment:
Some errors occurred, check the original file please.
Detailed exception are shown in console.

Reply:
Updated in https://github.com/MikeCoder/hexo-blog-encrypt/commit/862c24bcaf8b277c8b6956b23993a6533a48b52a